src/pdf/ShippingLabels/ShippingLabelsPdfButton.tsx

```diff
@@ -123,21 +124,22 @@ const getRequiredData = async (
             return { data: null, error };
         }
         const client = parcel.client;
+        const clientIsActive = parcel.client.is_active;
         parcelDataList.push({
             label_quantity: labelQuantity,
             parcel_id: parcelId,
             packing_slot: parcel.packing_slot?.name ?? "",
             collection_centre: parcel.collection_centre?.acronym ?? "",
             collection_datetime: formatDatetime(parcel.collection_datetime),
             voucher_number: parcel.voucher_number ?? "",
-            full_name: client.full_name,
-            phone_number: client.phone_number,
-            address_1: client.address_1,
-            address_2: client.address_2,
-            address_town: client.address_town,
-            address_county: client.address_county,
-            address_postcode: client.address_postcode,
-            delivery_instructions: client.delivery_instructions,
+            full_name: clientIsActive ? client.full_name ?? "" : displayNameForDeletedClient,
+            phone_number: clientIsActive ? client.phone_number ?? "" : "-",
```

Comment:
Is hyphen only used for inactive clients? - when do we use hyphen as supposed to the empty string.

Reply:
we only use the hyphen for inactive clients. we use the empty string if the client is active but the field is empty or null.

Reply:
Thanks, that makes sense!